.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -60,7 +63,10 @@ stages:
     pool:
       name: "$(BUILD_POOL_NAME_DEFAULT)"
     dependsOn:
-      - create_${{ parameters.name }}
+      - ${{ if and(eq(parameters.upgradeAzureIpam, ''), eq(parameters.upgradeScenario, false)) }}:

```

Comment:
Is this the only condition where we depend on cluster creation ? 
what if `parameters.upgradeScenario` is "" ? Can we add a table of the combination/expectation of these variables in the description ?